### ISSUE TITLE
Is there any plan to support linux?

### ISSUE THREAD

**Issue by GiftShower** [Sequal32/yourcontrols#245]
It would be really nice to play with others, but linux too. MSFS itself runs on linux, so I think it should be possible to make this plugin also available.

**Comment by BenedettiLucca** [comment]
The problem with making it run on linux is the current incompatibility between WebView2 and wine/proton. If this wasn't the case i think it would run smoothly on linux.

Would also love to see linux support.

**Comment by vsTerminus** [comment]
Problem 1 seems to be the use of https://docs.rs/tauri-web-view/latest/web_view/ which as BenedettiLucca mentioned uses webview2 when compiling for Windows and WINE has extremely limited support for webview2 right now (certain versions of WINE can install the runtime and not much else)

Problem 2 is the SimConnect bindings which seem to require a Windows Visual Studio environment to build.

So if you try to compile on Linux with a native target you fail to compile the simconnect dependency, and if you try to cross-compile for Windows even if you could build simconnect you'd just end up with a webview2 ui that doesn't work.

(I could also be misunderstanding how this all works)

So basically we're waiting for one of
1. The app moves away from webview2
2. WINE gets better support for webview2
3. The UI gets decoupled from the SimConnect components so the two can run separately, allowing the UI to be compiled natively for Linux
4. Support gets added for Command Line options allowing us to effectively bypass the UI entirely

Or maybe there are other/better options. I dunno. Unfortunately I don't think I'm able to do any of those things myself.

**Comment by Creative-Difficulty** [comment]
Number 3 would be awesome!

**Comment by GiftShower** [comment]
It's been a while since I made this, but I think currently making a Command Line would be the best option, since it looks like there is no possible way to compile targeting linux due to sinconnect.
